### DISCUSSION TITLE
OCR only the images?

### DISCUSSION THREAD

**Discussion by thibaultmol** [ocrmypdf/OCRmyPDF#1171] [Q&A] [answered]
Hi, I have pdf's that are exports from PowerPoint. They contain the actual slides as images, and then the notes for each slide as actual text in the pdf.

I would like only process the images and keep the existing text. I assume ocrmypdf is converting the entire page to an image and then doing ocr on the entire page?
If so: what options do i need to use to only have it ocr the actual images inside the pdf's.

**Comment by jbarlow83** [comment, marked as answer]
`--redo-ocr` just might do what you need despite the name. It attempts to hide existing text (if any) and then OCR whatever is leftover.

**Reply by thibaultmol** [reply]
Perfect! Exactly what I needed!
I'm so thankful for this tool existing, it's soo convenient and fast (thx to the auto thread count detection).

(Increased my sponsorship on OC but changed it from my personal account to company account)

**Reply by thibaultmol** [reply]
btw: search on readthedocs.io seems broken for me?
